nkms/blockchain/eth/sol/source/contracts/MinersEscrow.sol

```diff
@@ -356,46 +295,50 @@ contract MinersEscrow is Issuer {
     function withdraw(uint256 _value) public onlyTokenOwner {
         MinerInfo storage info = minerInfo[msg.sender];
         // TODO optimize
-        uint256 lockedTokens = Math.max256(calculateLockedTokens(msg.sender, 1),
-            getLockedTokens(msg.sender));
+        uint256 lockedTokens = Math.max256(getLockedTokens(msg.sender, 1),
```

Comment:
Eh.. why so?

Reply:
Is it in case we call after or before confirmActivity?

Reply:
In most cases in current period will have more locked tokens than all future periods. But if Ursula will increase stake (deposit or lock methods) than in next period will have maximum locked tokens